.github/workflows/benchmarks-nightly.yml

```diff
@@ -29,10 +29,9 @@ jobs:
     uses: ./.github/workflows/benchmarks-reusable.yml
     with:
         str_name: 'level_zero_v2'
-        unit: 'gpu'
+        preset: 'Full'
         pr_no: 0
-        bench_script_params: '--save baseline-v2'
+        bench_script_params: '--save Baseline_PVC_L0v2'
```

Comment:
did you mean Baseline_PVC_L0_V2 ?

Reply:
good catch, I'll change the config.js, I like `Baseline_PVC_L0v2` more.